types/artifacts.d.ts

```diff
@@ -213,6 +215,36 @@ declare global {
         requestId: string | null
       }
 
+      /** @see https://sourcemaps.info/spec.html#h.qz3o9nc69um5 */
+      export type RawSourceMap = {
+        /** File version (always the first entry in the object) and must be a positive integer. */
```

Comment:
```suggestion
        /** File version and must be a positive integer. */
```
(don't think we can/should commit to that unless it's really desirable)

Reply:
lol yeah this was a funny one. weird spec. didn't know if I should make any modifications so i copied wholesale.